chia/wallet/wallet_transaction_store.py

```diff
@@ -410,7 +415,7 @@ async def get_transactions_between(
 
     async def get_transaction_count_for_wallet(self, wallet_id) -> int:
         cursor = await self.db_connection.execute(
-            "SELECT COUNT(*) FROM transaction_record where wallet_id=?", (wallet_id,)
+            "SELECT COUNT(*) FROM transaction_record WHERE wallet_id=?", (wallet_id,)
```

Comment:
I imagine the `*` here isn't really relevant, right? this is counting rows, not columns

Reply:
I usually use count(1) as many others do, as it is faster to type on many keyboard layouts.
but it leads to the same results.